src/components/HotelDatePicker.vue

```diff
@@ -1,13 +1,16 @@
 <template>
```

Comment:
why  are you using `lang="pug"` in WIP but not here? On purpose?

Reply:
I'll remove this file once the WIP is done :)